src/glados/cli.py

```
@@ -1,193 +1,163 @@
+        if actual_checksum != expected_checksum:
+            progress.update(task_id, status="[bold red]Checksum failed")
+            Path.unlink(file_path)
+            return False
+        else:
+            progress.update(task_id, status="[bold green]OK")
+            return True
 
-    This function downloads the required model files from predefined URLs if they are missing
-    or have invalid checksums. It includes several reliability features:
+    except Exception as e:
+        progress.update(task_id, status=f"[bold red]Error: {str(e)}")
```

Ruff (0.11.9)

Use explicit conversion flag

Replace with conversion flag

(RUF010)